python_src/src/lamp_py/performance_manager/l0_rt_vehicle_positions.py

```diff
@@ -11,11 +11,12 @@
     start_time_to_seconds,
     add_static_version_key_column,
     add_parent_station_column,
-    remove_bus_records,
 )
 
 
-def get_vp_dataframe(to_load: Union[str, List[str]]) -> pandas.DataFrame:
+def get_vp_dataframe(
+    to_load: Union[str, List[str]], route_ids: List[str]
+) -> pandas.DataFrame:
```

Comment:
if the change is incorporated to call `rail_routes_from_timestamp` from inside of `get_vp_dataframe`, `route_ids` parameter could stay as defaulting to an empty list, so the option exists to manually pass `route_ids` into this and `get_tu_dataframe_chunks` for mocking